src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -43,14 +45,18 @@ export const buildATIPageTrackPath = ({
   campaigns,
 }) => {
   const href = getHref(platform);
+  const decodedHref = decodeURIComponent(href);
   const referrer = getReferrer(platform, origin, previousPath);
+  const campaignType = getCampaignType();
 
   // We use amp variable substitutes to get the href and referrer and these cannot be manipulated
   // For canonical, we have a requirement to encode the x5 and x6 value twice. Source issue: https://github.com/bbc/simorgh/pull/6593
   const x5Value = platform === 'amp' ? href : href && encodeURIComponent(href);
   const x6Value =
     platform === 'amp' ? referrer : referrer && encodeURIComponent(referrer);
 
+  const shouldIncludeMarketingString = process.env.SIMORGH_APP_ENV !== 'live';
```

Comment:
errm yeah, It's temporary as we just need to test it out on the test environment before enabling it. The reason I didn't make it a fully fledged toggle was because we usually access those from a react context with the useToggle hook, ie
```.jsx
useToggle('some-toggle');
```
@andrewscfc

Reply:
okay, makes sense, maybe worth making a note of this line in the final issue in case someone else picks this up and finds it doesn't work on live

Reply:
We'd need to create and link an issue in the final issue to test this on the test environment and based on the result on that create a follow up issue to fix forwards or enable it on live is if it's good. I'll coordinate this with @paruchurisilpa.